src/iFIT.Xamarin.ExoPlayer/iFIT.Xamarin.ExoPlayer.csproj

```diff
@@ -5,7 +5,7 @@
     <RootNamespace>Google.Android.ExoPlayer2</RootNamespace>
     <PackOnBuild>true</PackOnBuild>
     <PackageId>iFIT.Xamarin.ExoPlayer</PackageId>
-    <PackageVersion>2.13.3.4</PackageVersion>
+    <PackageVersion>2.13.3.5-buff216</PackageVersion>
```

Comment:
So, don't change anything about this PR right now.  If everything else is exactly as we want it, then we might merge this to another temporary branch in case there is more exoplayer work we need to do, but we won't want to merge to main or release with a pre-release package version.  We'll want this to ultimately be `2.13.3.5` (but we want to leave it as prerelease until we are satisfied with things downstream).